sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -206,7 +206,8 @@ public class EventProcessorClient : EventProcessor<EventProcessorPartition>
         ///
         ///   The exceptions surfaced to this method may be fatal or non-fatal; because the processor may not be able to accurately predict whether an
         ///   exception was fatal or whether its state was corrupted, this method has responsibility for making the determination as to whether processing
-        ///   should be terminated or restarted.  The method may do so by calling Stop on the processor instance and then, if desired, calling Start on the processor.
+        ///   should be terminated or restarted.  If desired, it is safe for this method to manage the processor by calling <see cref="StopProcessingAsync" />
```

Comment:
```suggestion
        ///   should be terminated or restarted.  If desired, this can be done safely by calling <see cref="StopProcessingAsync" />
```

Reply:
Manually applied to both processors.